packages/examples/tsconfig.json

```diff
@@ -2,6 +2,7 @@
     "extends": "../../tsconfig.json",
     "compilerOptions": {
         "outDir": "dist",
+        "module": "commonjs",
```

Comment:
Question: I do not fully understand why this change is needed? Can't we live with the default? 
I did read your comment in the PR description, but it does not explain to me the rational ...

Reply:
I see. Thanks for the clarification!